automarker_2/automarker_app/lib/steps.py

```diff
@@ -639,7 +642,12 @@ class PythonDoRequirementsTxtInstall(Step):
     name = "pip install requirements.txt"
 
     def run(self, project_uri, clone_dir_path, self_test, config, fail_fast):
-        command = f"{clone_dir_path/'automarker_venv'/'bin'/'pip'} install -r {clone_dir_path/'requirements.txt'}"
+        if os.name == "nt":
+            python_executable_path = clone_dir_path / 'automarker_venv' / 'Scripts' / 'pip'
```

Comment:
Is this a python executable path or pip?

Reply:
if we want to stick to using python executable then we have to adapt it to "executable -m pip"